src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -506,13 +587,60 @@ public static void hoverInAppServerXML(RemoteRobot remoteRobot, String hoverTarg
             // Check for "Fetching Documentation" message indicating there is a delay in getting hint
             // allow some time for the LS hint to appear in the popup
             if (remoteString.contains("Fetching Documentation")) {
-                TestUtils.sleepAndIgnoreException(2);
+                // this will allow for a 15 sec delay potentially to allow doc hint window to populate
+                TestUtils.sleepAndIgnoreException(5);
             } else {
                 break;
             }
         }
     }
 
+    public static void jitterCursor(EditorFixture editor, int pointX, int pointY) {
+
+        String jitterScript = "const x = %d;" +
+                "const y = %d;" +
+                "java.util.List.of(5, 20, 5, 15).forEach((i)=> {" +
+                "const point = new Point(x + i, y);" +
+                "robot.moveMouse(component, point);})";
+
+        // run the jitter mouse script remotely in the idea
+        editor.runJs(String.format(jitterScript, pointX, pointY));
+    }
+
+    public static void insertEnvCfgInAppCfgFile(RemoteRobot remoteRobot, String projectName, String fileName, String envCfgNameSnippet, String envCfgNameChooserSnippet, String envCfgValueSnippet) {
```

Comment:
suggest to rename to `insertConfigInAppCfgFile` or `insertConfigInConfigFile`
...and maybe rename the "envCfg" prefixes on the params to "config"

Reply:
These suggestions still apply with the latest changes